docs/ref/contrib/admin/index.txt

```diff
@@ -2369,6 +2372,17 @@ The ``InlineModelAdmin`` class adds:
     inline forms. For example, this may be based on the model instance
     (passed as the keyword argument ``obj``).
 
+.. method:: InlineModelAdmin.has_add_permission(request, obj=None)
+
+   Should return ``True`` if adding a child object inline is permitted,
+   ``False`` otherwise. ``obj`` is the parent object being edited. If ``obj``
+   is ``None``, should return ``True`` or ``False`` when adding a new parent
+   object.
```

Comment:
I don't understand the _If `obj` is `None`_ case. I think it needs expanding slightly. 

@icu0755 was right on the original PR: beyond the addition here, there are no docs for the `InlineModelAdmin` versions of these methods. Whilst the signature is the same, the semantics are different. Perhaps that is a different ticket, or commit at least (to add  `has_change_permission` and `has_delete_permission`).

Reply:
> I don't understand the If obj is None case. I think it needs expanding slightly.

I've updated this to the following. Let me know if you think it is an improvement. I'm open to additional wording suggestions.

> Should return `True` if adding a child object inline is permitted, `False` otherwise. `obj` is the parent object being edited. When adding a new parent object, `obj` is `None`.